src/components/Timeout/ModalPopup.js

```diff
@@ -28,54 +39,54 @@ const ModalPopup = ({ countdownTime, setIdle, logoff, timedOut }) => {
     return () => clearInterval(timer);
   }, [countdownTime, timedOut]);
 
+  const handleContinue = () => {
+    setIdle(false);
+    reset();
+  };
+
   return (
-    <div className={classes.modal_background}>
-      <FocusTrap open isEnabled={() => true} getDoc={() => document}>
-        <article className={classes.modal}>
-          <div className={classes.modal__center_div}>
-            <div className={classes.modal__center_div__content}>
-              <h1 className={classes.modal__center_div__content__p_head}>
-                Your online session
-              </h1>
-              <p className={classes.modal__center_div__content__p_head}>
-                will expire soon
-              </p>
-              <p className={classes.modal__center_div__content__p_timer}>
-                {`${minutes} min ${seconds < 10 ? '0' : ''}${seconds} seconds`}
-              </p>
-              <p className={classes.modal__center_div__content__p_text}>
-                Choose continue to keep working or log off.
-              </p>
-            </div>
-            <div className={classes.modal__center_div__buttons}>
-              <Button
-                {...{
-                  className: classes.modal__center_div__buttons__button_logoff,
-                  onClick: logoff,
-                  onKeyPress: logoff,
-                  'aria-label': 'log off',
-                  tabIndex: 0,
-                }}
-              >
-                Log Off
-              </Button>
-              <Button
-                {...{
-                  className:
-                    classes.modal__center_div__buttons__button_continue,
-                  onClick: () => setIdle(false),
-                  onKeyPress: () => setIdle(false),
-                  'aria-label': 'continue',
-                  tabIndex: 0,
-                }}
-              >
-                Continue
-              </Button>
-            </div>
+    <FocusTrapOverlay open={open}>
+      <Paper
```

Comment:
can we switch this to a Dialog component? like this below.. thanks

`<Dialog
        {...{
          open: true,
          disableAutoFocus: true,
          role: 'alertdialog',
          'aria-label': 'Session timeout modal',
          'aria-modal': true,
          className: modal,
        }}
      >`

Reply:
i found a fix for the error on scenarios where there are other modal popup in the dom like the profile drop down